src/utils/rest-api.js

```diff
@@ -884,6 +884,80 @@ export function fetchSensitivityAnalysisResult(studyUuid, currentNodeUuid) {
     });
 }
 
+export function startShortCircuitAnalysis(studyUuid, currentNodeUuid) {
+    console.info(
+        'Running short circuit analysis on ' +
```

Comment:
Could use backticks string: `Running short circuit analysis on ${studyUuid] and node ${currentNodeUuid} ...`

Reply:
done